common/src/main/java/org/apache/nemo/common/ir/vertex/executionproperty/ResourceLambdaProperty.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.nemo.common.ir.vertex.executionproperty;
+
+import org.apache.nemo.common.ir.executionproperty.VertexExecutionProperty;
+
+/**
+ * Maintainer: Gao Zhiyuan<alapha23@gmail.com>.
+ * Resource property supporting Lambda Pass
+ */
+public final class ResourceLambdaProperty extends VertexExecutionProperty<ResourceLambdaProperty.Value> {
```

Comment:
why don't you add a `LAMBDA` variable in `ResourcePriorityProperty` class? we don't need to create another class for the resource lambda property.